plugins/ssh-jump.yaml

```diff
@@ -3,28 +3,28 @@ kind: Plugin
 metadata:
   name: ssh-jump
 spec:
-  version: "v0.5.0"
+  version: "v0.6.0"
   homepage: https://github.com/yokawasa/kubectl-plugin-ssh-jump
   platforms:
   - selector:
       matchExpressions:
       - {key: os, operator: In, values: [darwin, linux]}
-    uri: https://github.com/yokawasa/kubectl-plugin-ssh-jump/archive/0.5.0.zip
-    sha256: "3b7091d90a1ad082c078a406f7b455a9f5686996d3c273efe589c5ae4c445bd8"
+    uri: https://github.com/yokawasa/kubectl-plugin-ssh-jump/archive/0.6.0.zip
+    sha256: "a24149bb1b1ccba91ae87c86d21625adf9e110b9e81a0ab58ec84343e0f99ede"
     files:
     - from: "./*/kubectl-ssh-jump"
       to: .
     - from: "./*/LICENSE"
       to: .
     bin: "./kubectl-ssh-jump"
-  shortDescription: A kubectl plugin to SSH into Kubernetes nodes using a SSH jump host Pod
+  shortDescription: A kubectl plugin to access Kubernetes nodes or remote services using a SSH jump Pod
```

Comment:
you should strive to keep this short <50 chars as it will be truncated.

there are many redundant terms here like `A kubectl plugin to` or `Kubernetes`. Just say:

> Access nodes or services using SSH jump pod

Reply:
@ahmetb thank you so much for the suggestion. I've shorten the description https://github.com/kubernetes-sigs/krew-index/pull/1961/commits/44c32f3f640ad502cd64d968d1c8896a18c54f9f